packages/server/src/integrations/utils.ts

```diff
@@ -143,11 +143,21 @@ export function isIsoDateString(str: string) {
   return d.toISOString() === str
 }
 
+function shouldCopyRelationship(column: { type: string, tableId?: string }, tableIds: [string]) {
```

Comment:
Can we add some doc to both these new functions? On the surface all looks good but I'm really not sure why / what they are for 😅

Reply:
Thank you! much easier to understand now